sagemaker-debugger/pytorch_iterative_model_pruning/iterative_model_pruning.ipynb

```diff
@@ -0,0 +1,640 @@
+{
```

Comment:
<p>[Grammar] </p><ol><li><code>However, it may ...</code></li><li><code><s>shrinkes</s></code> --&gt; <code>shrinks</code> </li></ol><p> </p><p>[Misc]</p><p>May be link to the AWS documentation of SageMaker Experiments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1057/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>